### PR TITLE
Added reproducible for rskj/6.5.0-arrowhead

### PR DESCRIPTION
**Pull request by Vovchyk**
checksums:

```
294f99ec76befa7d9ef18eca28d0c884ceb463ce855ef23ddb7a476e551ba191  rskj-core-6.5.0-ARROWHEAD-all.jar
ec4ea3143fb95fbf878b24c8d00ebe5994c4586ecf92930743b26dc77395c17a  rskj-core-6.5.0-ARROWHEAD-sources.jar
7721f59632d9ff825b0bb23cf52fa3eff347852068a03d792ce18d0deae547ae  rskj-core-6.5.0-ARROWHEAD.jar
16283f4aebbf5b5b35c63f098d81fb0d2aeb5e9fc3be51128ab300333127077f  rskj-core-6.5.0-ARROWHEAD.module
ee4f7e684044302e3819f8c50d9b85af5b3391d14e6ed6110b98e8387a1b2837  rskj-core-6.5.0-ARROWHEAD.pom
```

### CODE REVIEW

**Review by marcos-iov — APPROVED** [review]
```
294f99ec76befa7d9ef18eca28d0c884ceb463ce855ef23ddb7a476e551ba191  rskj-core-6.5.0-ARROWHEAD-all.jar
ec4ea3143fb95fbf878b24c8d00ebe5994c4586ecf92930743b26dc77395c17a  rskj-core-6.5.0-ARROWHEAD-sources.jar
7721f59632d9ff825b0bb23cf52fa3eff347852068a03d792ce18d0deae547ae  rskj-core-6.5.0-ARROWHEAD.jar
16283f4aebbf5b5b35c63f098d81fb0d2aeb5e9fc3be51128ab300333127077f  rskj-core-6.5.0-ARROWHEAD.module
ee4f7e684044302e3819f8c50d9b85af5b3391d14e6ed6110b98e8387a1b2837  rskj-core-6.5.0-ARROWHEAD.pom 
```

File: rskj/6.5.0-arrowhead/README.md
Comment (julia-zack) [inline rskj/6.5.0-arrowhead/README.md:1]: 294f99ec76befa7d9ef18eca28d0c884ceb463ce855ef23ddb7a476e551ba191  rskj-core-6.5.0-ARROWHEAD-all.jar
ec4ea3143fb95fbf878b24c8d00ebe5994c4586ecf92930743b26dc77395c17a  rskj-core-6.5.0-ARROWHEAD-sources.jar
7721f59632d9ff825b0bb23cf52fa3eff347852068a03d792ce18d0deae547ae  rskj-core-6.5.0-ARROWHEAD.jar
16283f4aebbf5b5b35c63f098d81fb0d2aeb5e9fc3be51128ab300333127077f  rskj-core-6.5.0-ARROWHEAD.module
ee4f7e684044302e3819f8c50d9b85af5b3391d14e6ed6110b98e8387a1b2837  rskj-core-6.5.0-ARROWHEAD.pom

**Review by julia-zack — APPROVED** [review]

**Review by nathanieliov — APPROVED** [review]
```
~/dev/workspace/iovlabs/projects/reproducible-builds/rskj/6.5.0-arrowhead arrowhead-650 ?4 -------------------------------------------------------------------- 2m 41s 15:39:11
> docker run --rm rskj/6.5.0-arrowhead sh -c 'sha256sum * | grep -v javadoc.jar'
294f99ec76befa7d9ef18eca28d0c884ceb463ce855ef23ddb7a476e551ba191  rskj-core-6.5.0-ARROWHEAD-all.jar
ec4ea3143fb95fbf878b24c8d00ebe5994c4586ecf92930743b26dc77395c17a  rskj-core-6.5.0-ARROWHEAD-sources.jar
7721f59632d9ff825b0bb23cf52fa3eff347852068a03d792ce18d0deae547ae  rskj-core-6.5.0-ARROWHEAD.jar
16283f4aebbf5b5b35c63f098d81fb0d2aeb5e9fc3be51128ab300333127077f  rskj-core-6.5.0-ARROWHEAD.module
ee4f7e684044302e3819f8c50d9b85af5b3391d14e6ed6110b98e8387a1b2837  rskj-core-6.5.0-ARROWHEAD.pom
```

**Comment by jeremy-then** [comment]
294f99ec76befa7d9ef18eca28d0c884ceb463ce855ef23ddb7a476e551ba191  rskj-core-6.5.0-ARROWHEAD-all.jar
ec4ea3143fb95fbf878b24c8d00ebe5994c4586ecf92930743b26dc77395c17a  rskj-core-6.5.0-ARROWHEAD-sources.jar
7721f59632d9ff825b0bb23cf52fa3eff347852068a03d792ce18d0deae547ae  rskj-core-6.5.0-ARROWHEAD.jar
16283f4aebbf5b5b35c63f098d81fb0d2aeb5e9fc3be51128ab300333127077f  rskj-core-6.5.0-ARROWHEAD.module
ee4f7e684044302e3819f8c50d9b85af5b3391d14e6ed6110b98e8387a1b2837  rskj-core-6.5.0-ARROWHEAD.pom

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Pull request merged**